src/components/PageHeadingFlowRun.vue

```diff
@@ -1,26 +1,77 @@
 <template>
-  <page-heading v-if="flowRun" class="page-heading-flow-run" :crumbs="crumbs">
-    <template #after-crumbs>
+  <div v-if="flowRun" class="page-heading-flow-run">
+    <page-heading class="page-heading-flow-run__heading" :crumbs="crumbs">
+      <template #after-crumbs>
+        <div v-if="flowRun.tags && flowRun.tags.length > 0" class="state-list-item__tags">
+          <p-tag-wrapper v-bind="{ tags: flowRun.tags }" />
+        </div>
+      </template>
+      <template #actions>
+        <template v-if="media.sm">
+          <FlowRunPauseButton :flow-run="flowRun" />
+          <FlowRunResumeButton :flow-run="flowRun" />
+          <FlowRunRetryButton :flow-run="flowRun" />
+          <FlowRunCancelButton :flow-run="flowRun" />
+        </template>
+        <FlowRunMenu :flow-run-id="flowRun.id" :show-all="!media.sm" @delete="emit('delete')" />
+      </template>
+    </page-heading>
+    <div class="page-heading-flow-run__meta">
       <StateBadge :state="flowRun.state" />
-    </template>
-    <template #actions>
-      <template v-if="media.sm">
-        <FlowRunPauseButton :flow-run="flowRun" />
-        <FlowRunResumeButton :flow-run="flowRun" />
-        <FlowRunRetryButton :flow-run="flowRun" />
-        <FlowRunCancelButton :flow-run="flowRun" />
+      <FlowRunStartTime :flow-run="flowRun" />
+      <DurationIconText :duration="flowRun.duration" />
+      <template v-if="!isPending">
+        <FlowRunTaskCount :tasks-count="taskRunsCount">
+          <template #default="{ count }">
+            {{ count }} task {{ toPluralString('run', count) }}
+          </template>
+        </FlowRunTaskCount>
```

Comment:
We don't use this `FlowRunTaskCount` component anywhere without using this custom slot that I can find. Can you just update the component to do this by default?